app/src/highlighter/index.ts

```diff
@@ -172,6 +177,8 @@ const extensionModes: ReadonlyArray<IModeDefinition> = [
     install: () => import('codemirror/mode/shell/shell'),
     mappings: {
       '.sh': 'text/x-sh',
+      '.bat': 'text/x-sh',
+      '.cmd': 'text/x-sh',
```

Comment:
`.bat` and `.cmd` are a whole different language from that of shell script and that language is not supported in codemirror at this time, see https://github.com/codemirror/codemirror5/issues/5497

Reply:
Removed .bat and .cmd from this pull request